src/main/features/core/tray.js

```diff
@@ -61,14 +73,17 @@ Emitter.on('playback:isStopped', () => {
   appIcon.setImage(getAppIconFileName());
 });
 
-const setContextMenu = () => {
+const setContextMenu = (song) => {
```

Comment:
Since `4.0.0` we have been referring to the current playing track as "track" internally to match up with other libraries we use and for general consistency

s/song/track